test/system/505-networking-pasta.bats

```diff
@@ -681,3 +681,11 @@ function teardown() {
     sleep 1
     ! ps -p $(cat "${pidfile}") && rm "${pidfile}"
 }
+
+### Options ####################################################################
+@test "podman networking with pasta(1) - Unsupported protocol in port forwarding" {
+    local port=$(random_free_port "" "" tcp)
+
+    run_podman 126 run --net=pasta -p "${port}:${port}/sctp" $IMAGE true
+    is "$output" "Error: can't forward protocol: sctp"
```

Comment:
```suggestion
    is "$output" "Error: .*can't forward protocol: sctp"
```
The check is failing with podman-remote, see https://api.cirrus-ci.com/v1/artifact/task/6684547220242432/html/sys-remote-fedora-38-rootless-host-boltdb.log.html

Reply:
Ah, thanks.  I saw the failure but didn't know how to dig far enough into the logs to find a helpful error message.